azure/packages/test/end-to-end-tests/.npmignore

```diff
@@ -2,3 +2,5 @@ nyc
 *.log
 **/*.tsbuildinfo
 **/_api-extractor-temp/**
+src/test
```

Comment:
Is making this package only contain the packageVersion.ts file really the right approach?

It seems like an end to end tests folder should either make a package containing the tests, or not making a package at all. Given that this package is not marked as private, I assume it has users and this might break them? I don't see it published on npm though.

Reply:
Good catch, Alex pointed out the same thing. I've removed this at the cost of adding a warning back.

Reply:
Bit of follow-up, we can probably remove that warning by adding the full name of this package to the exclusions during the check (since it doesn't match the current ones of `@<scope>/test-*`), [here](https://github.com/microsoft/FluidFramework/blob/4ccc4723872b15d7fde5a815d6f670394a8c416f/build-tools/packages/build-tools/src/fluidBuild/fluidPackageCheck.ts#L475-L482).